python/golem_task_api/testutils/tasklifecycleutil.py

```diff
@@ -0,0 +1,172 @@
+import shutil
+from pathlib import Path
+from typing import Callable, List, Tuple
+
+from async_generator import asynccontextmanager
+
+from golem_task_api import (
+    TaskApiService,
+    constants,
+    ProviderAppClient,
+    RequestorAppClient,
+)
+
+
+class TaskLifecycleUtil:
+    def __init__(self, work_dir: Path) -> None:
+        print('workdir:', work_dir)
```

Comment:
I believe most people don't like random prints in library code so I'd suggest removing this one.